geth/node/node.go

```diff
@@ -92,7 +92,7 @@ func defaultEmbeddedNodeConfig(config *params.NodeConfig) *node.Config {
 		Version:           config.Version,
 		P2P: p2p.Config{
 			NoDiscovery:      !config.Discovery,
-			DiscoveryV5:      true,
+			DiscoveryV5:      config.Discovery,
```

Comment:
it wouldn't make sense for us to run both

maybe it is better to make it always false, until we will find a reason to run discovery and settle on a preferred version?

Reply:
Looking at how this part is handled, I thought that it is expected to use both: see `p2p/server.go#L455`.

Reply:
455? don't see anything related there, i see that it can share a connection with regular discovery https://github.com/ethereum/go-ethereum/blob/master/p2p/server.go#L459-L474 (sconn) .  but i think that running both is a migration

Reply:
Yeah, perhaps you're right.
Even though, I'd keep it configurable as is.
It is weird that we configure one but not the other.